readme.md

```diff
@@ -436,7 +436,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Bots](https://github.com/hackerkid/bots) - Building bots.
 - [Site Reliability Engineering](https://github.com/dastergon/awesome-sre)
 - [Empathy in Engineering](https://github.com/KimberlyMunoz/empathy-in-engineering) - Building and promoting more compassionate engineering cultures.
-
+- [DTrace](https://github.com/xen0l/awesome-dtrace) - DTrace - Dynamic tracing framework.
```

Comment:
I've already commented about this... => https://github.com/sindresorhus/awesome/pull/701#discussion_r69030389